src/PositionVelocityTime.jl

```diff
@@ -23,24 +27,56 @@ module PositionVelocityTime
         carrier_phase::CP = 0
     end
 
+    """
+    DOP, stores the Dilution of Precision values
+    """
+    @with_kw mutable struct DOP
```

Comment:
Does it need to be mutable?
A non-mutable struct would be more efficient and also makes it easier to reason about.

Reply:
No, you are right.